src/pages/Photos/components/Sort/Sort.scss

```diff
@@ -0,0 +1,121 @@
+/* stylelint-disable no-descending-specificity */
+.sort {
+  $this: &;
+
+  position: relative;
+
+  &__label {
+    display: none;
+
+    @include tablet {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  &__title {
+    margin-right: 12px;
+    color: $color-medium-grey;
+  }
+
+  &__variable {
+    margin-right: 8px;
+    font-size: 14px;
+    color: $color-dark-grey;
+  }
+
+  &__select-popup-btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: 8px;
+    outline: 1px solid $color-grey-light-200;
+    border-radius: 6px;
+    padding: 16px;
+
+    svg {
+      @include icon(16px,21px,$color-black);
+    }
+
+    &:hover {
+      background-color: $color-grey-light-100;
+
+      svg {
+        transform: rotate(180deg);
+      }
+    }
+
+    @include tablet {
+      display: none;
+    }
+  }
+
+  &__btn-action {
+    &:hover {
+      #{$this}__arrow-icon {
+        @include icon(8px,12px,$color-dark-grey);
+      }
+    }
+
+    &:active {
+      #{$this}__arrow-icon {
+        transform: translateY(3px);
+      }
+    }
+  }
+
+  &__arrow-icon {
+    @include icon(8px,12px,$color-medium-grey);
+  }
+
+  &__popup {
+    position: absolute;
+    right: 0;
+    top: 64px;
+    z-index: $layer-popup;
+    margin: 0;
+    border-radius: 6px;
+    padding: 8px;
+    width: 224px;
+    box-shadow: 1px 4px 16px rgb(180 173 189 / 15%);
+    background-color: $color-white;
+
+    @include tablet {
+      right: 0;
+      top: 32px;
+    }
+  }
+
+  &__popup-list {
+    @include reset-list;
+
+    width: 100%;
+  }
+
+  &__popup-item {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 12px 10px;
```

Comment:
please double check the spacings on the popup, as visually they don't match the design

Reply:
I changed the margins, but there is confusion in the design.

Mobile padding 
![Jbd_XNzZ6l8](https://user-images.githubusercontent.com/70577403/236008108-3eb198e8-1656-4ca0-b2c5-9bbb735b322d.jpg)
![lPi90LNlHbY](https://user-images.githubusercontent.com/70577403/236008123-8c45e86c-9273-41db-848a-09593b7b79a4.jpg)

Desktop
![0tG-YvgE7Uo](https://user-images.githubusercontent.com/70577403/236008255-4c524c28-6cdf-45f1-8807-0dd7db343d5a.jpg)

![VB8j4biHlTM](https://user-images.githubusercontent.com/70577403/236008264-13857173-e220-442a-a196-dded4fc63d36.jpg)